### ISSUE TITLE
update copilot-version

### ISSUE THREAD

**Issue by bbigras** [copilot-emacs/copilot.el#304]
btw, new versions don't seem to use `bin/copilot-node-server` anymore.

**Comment by jcs090218** [comment]
Yeah, you can now use `M-x copilot-install-server` to install the server.

**Comment by bbigras** [comment]
I mean that this version is outdated:

https://github.com/copilot-emacs/copilot.el/blob/733bff26450255e092c10873580e9abfed8a81b8/copilot.el#L147-L148

but the latest copilot-node-server version doesn't have the `bin/copilot-node-server` file anymore, so `copilot.el` would need to have some changes.

**Comment by jcs090218** [comment]
The command line has been changed to `node copilot/dist/agent.js --stdio` based on their README page.

**Comment by jcs090218** [comment]
Can you try #305? 🤔
